.devcontainer/bootstrap.sh

```diff
@@ -25,7 +25,7 @@ sudo apt-get install -y kubectl
 curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
 
 # Install Spin
-mkdir spininstall && cd spininstall
+mkdir -p spininstall && cd spininstall
```

Comment:
nit: this should be deleting the folder if exists and creating it fresh to avoid inconsistent intermediate states in the folder... or rebuild the container if we stop the container's post create scripts in-between.

Reply:
Please rebase from main which has changes to recreate the folder if it exists already.